### ISSUE TITLE
Error 400 when trying to create subscription

### ISSUE THREAD

**Issue by madhavsankar** [SharePoint/sp-dev-docs#5622]
## Category

- [x] Bug

## Describe the bug
As per https://docs.microsoft.com/en-us/sharepoint/dev/apis/webhooks/overview-sharepoint-webhooks,

Receiving a 400 Bad request error when trying to create subscription:
note: I have removed the site_id and list_id from the error and the description.
Error:
{
    "error": {
        "code": "BadRequest",
        "message": "The value 'https://graph.microsoft.com/v1.0/sites/<site_id>/lists/<list_id_of_shared_documents>' of property 'resource' must be a relative URI.",
        "innerError": {
            "request-id": "bff8ab9e-36ab-4cdb-9367-7de16b7a9925",
            "date": "2020-04-23T08:10:46"
        }
    }
}

## Steps to reproduce
Sent the following request from the Graph Explorer (signed in using the sharepoint admin credentials)

POST request to https://graph.microsoft.com/v1.0/sites/<site_id>/lists/<list_id>/subscriptions

Body:
{"resource":"https://graph.microsoft.com/v1.0/sites/<site_id>/lists/<list_id>","notificationUrl":"<azure_functions_url>","expirationDateTime":"2020-04-27T16:17:57+00:00"}

## Expected behavior
Should receive Status code 201

**Comment by JarbasHorst** [comment]
Hello @madhavsankar 

according to [this ](https://docs.microsoft.com/en-us/sharepoint/dev/apis/webhooks/overview-sharepoint-webhooks) documentation Microsoft Graph is not required for creating a new subscription. If you check this [link](https://docs.microsoft.com/en-us/sharepoint/dev/apis/webhooks/lists/overview-sharepoint-list-webhooks), you will notice that SharePoint REST API is used to create the subscription. Please, double check the "[Create a new subscription](https://docs.microsoft.com/en-us/sharepoint/dev/apis/webhooks/lists/create-subscription)" documentation in order to correctly set up your request.

Basically, your POST request should be something like this: https://contoso.sharepoint.com/_api/web/lists('5C77031A-9621-4DFC-BB5D-57803A94E91D')/subscriptions instead of https://graph.microsoft.com/v1.0/sites/<site_id>/lists/<list_id_of_shared_documents>

Regards,
Jarbas

**Comment by JeremyKelley** [comment]
If you do want to use Graph notifications you would actually use the top level /subscriptions endpoint and post the Drive ID as a resource in the body.  So you'd post to https://graph.microsoft.com/v1.0/subscriptions and then in the body you'll have a bunch of parameters including the resource path.

The resource path for a drive looks something like this:
"sites/XXXXXXX.sharepoint.com,635aa1ec-d549-4969-bd29-326bbaa7e8ff,9a507f58-9dfd-4925-858a-8e1c43313db1/drives/b!7KFaY0nVaUm9KTJruqfo_1h_UJr9nSVJhYqOHEMxPbFn10xGquBmRZXQZmbxwtYJ/root"

This page should also help:  https://docs.microsoft.com/en-us/graph/webhooks?context=graph%2Fapi%2F1.0&view=graph-rest-1.0
